spec/features/directors/index_spec.rb

```diff
@@ -44,19 +45,25 @@
       it 'displays a link called Films Index' do
         visit '/directors'
 
-        expect(page).to have_link("Films Index", :href=>"/films")
+        expect(page).to have_link("Films Index", href: "/films")
       end
 
       it 'displays a link called Directors Index' do
         visit '/directors'
 
-        expect(page).to have_link("Directors Index", :href=>"/directors")
+        expect(page).to have_link("Directors Index", href: "/directors")
```

Comment:
Ah, nice. I guess which way doesn't matter. It is just a matter of preference and cutting down on keystrokes.